### PR TITLE
Revert the op reentry workaround for the stress tests.

### PR DESCRIPTION
**Pull request by andre4i**
## Description

Reverting https://github.com/microsoft/FluidFramework/commit/aa77a0f3dfdcd3d79abac2e951dcdd3038fdb847 as it is suspected to be causing a race condition.

### CODE REVIEW

File: packages/test/test-service-load/src/loadTestDataStore.ts [hunk 1]
```diff
@@ -233,16 +233,10 @@ export class LoadTestDataStoreModel {
...
-				() => {
-					if (taskId === this.taskId && this.taskStartTime !== 0) {
-						this.dir.set(taskTimeKey, this.totalTaskTime);
-						this.taskStartTime = 0;
-					}
-				},
-				(error) =>
-					this.logger.sendErrorEvent({ eventName: "TaskManager_OnValueChanged" }, error),
-			);
+			if (taskId === this.taskId && this.taskStartTime !== 0) {
```
Comment (vladsud) [inline packages/test/test-service-load/src/loadTestDataStore.ts:236]: I'm not sure I agree with a direction in the PR.
We know that telemetry actually captures wrong-doing - it maybe not due to your changes, but it does good stuff (brings our attention to the problem). And we identified already one case that we can fix.
I'd love to see addressing it and re-evaluating if we can undo changes in this file but keep an error as an error.
At least for starters. And re-evaluate.

Basically, we have a problem, and having errors helps keep the problem active.

**Comment by andre4i** [comment]
This was a red herring. The real issue was the volume of ops sent by the stress test clients.
